src/features/_links.scss

```diff
@@ -0,0 +1,108 @@
+/* Warn when opening a link in a new tab or window */
+a[target = "_blank"] {
+  @extend %alertWarning;
+
+  &::after {
+    @extend %contentWarning;
+    @include contentMessage(warning, "Links that open in a new tab or window should be communicated to the user, ensure the user knows of this behavior.");
+  }
+}
+
+/* Invalid HTML nested inside <a> */
+a {
+  audio[controls] {
+    @extend %alertError;
+
+    &::after {
+      @include contentMessage(error, "Ensure that <audio> with controls attribute is not a child of <a> as it is an invalid HTML.");
```

Comment:
These are SO MUCH CLEARER 👏
Love it when the comments can be entirely removed because they just aren't needed any more.